container-images/vulkan/Containerfile

```diff
@@ -0,0 +1,23 @@
+FROM quay.io/ramalama/ramalama:latest
+
+RUN /usr/bin/python3 --version
+
+RUN git clone https://github.com/ggerganov/llama.cpp && \
+    cd llama.cpp && \
+    git reset --hard ${LLAMA_CPP_SHA} && \
```

Comment:
@rhatdan do you know if ARGS like LLAMA_CPP_SHA and WHISPER_CPP_SHA get inherited from parent container images?

Reply:
I'm assuming they don't, but I don't know

Reply:
Only if they are set as Environment variables, which currently they are not.  If we want this to be set we could do it via environment in ramalama Containerfile

env LLAMA_CPP_SHA=${LLAMA_CPP_SHA}

Reply:
Trying it out now.

Reply:
Works: https://github.com/containers/ramalama/pull/355

Reply:
Cool that's better, I'll merge yours